docs/guides/file-system-based-automated-bundle-generation.md

```diff
@@ -0,0 +1,116 @@
+# File System Based Automated Bundle Generation
```

Comment:
1. I think it's more correct to say "file-system-based automated bundle generation" (both here and in `README.md` line 54).
2. Why not just "automated bundle generation" in the first place?